src/components/MyAccount/Settings/AccountSettingsButtons.tsx

```diff
@@ -0,0 +1,55 @@
+import { Icon, Button } from "@nypl/design-system-react-components"
+import type { Dispatch } from "react"
+
+interface AccountSettingsButtonsPropsType {
+  currentlyEditing: boolean
+  setCurrentlyEditing: Dispatch<React.SetStateAction<boolean>>
+}
+
+const AccountSettingsButtons = ({
+  currentlyEditing,
+  setCurrentlyEditing,
+}: AccountSettingsButtonsPropsType) => {
+  const toggleCurrentlyEditing = () => setCurrentlyEditing(!currentlyEditing)
+
+  const clearUpdate = () => {
```

Comment:
It's hard to tell because it's not testable today, but why not directly pass the value to the behavior of the button? So always set the value to false for the cancel button and true for the edit button?

I'm assuming that the toggle works because you can't access the edit button if the value is not already false, so calling `setCurrentlyEditing` will always set it to true. But perhaps being explicit will help read through this.